src/sound/ogg_playback.rs

```diff
@@ -14,13 +14,15 @@ pub(crate) struct OggPlaybackEngine {
     playing_intro: bool,
```

Comment:
we should use bitflags instead of two bools, see what we do for bitflags already in other places